docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -1497,6 +1479,63 @@ simulants living in military group quarters we will set their employer
 to Military, and ensure that their address and zip code match their
 employer_address and employer_zipcode.
 
+.. _census_prl_perturbation:
+
+2.3.16 Perturbation
+~~~~~~~~~~~~~~~~~~~
+
+When we sample from the ACS PUMS to generate new simulants, we are using the
+empirical joint distribution of all attributes derived from ACS
+variables.
+This allows us to replicate correlations of arbitrary complexity, but also
+causes us to over-fit to sampling variation in the ACS PUMS, which would not
+be present in the full population.
+For example, just because there are no individuals in the ACS PUMS with a
+particular combination of demographic attributes, that does not mean we would
+expect to find 0 such people in the entire US.
+For PRL it is particularly important not to generate simulants that are more similar
+to one another than would be expected in a real population, which
+would make matching unrealistically difficult.
+
+To decrease similarity without assuming total independence between attributes,
+we perturb values at sampling time.
+Specifically, we perturb the age and PUMA columns.
+These are the columns with many possible values, where sampling noise is
+likely to be a significant concern even at the substantial sample size (>15 million)
+of the ACS PUMS.
+
+In different components of the simulation, we sample different entity types from the PUMS:
+households, group quarters people, or non-GQ people (individuals living in households).
+The perturbation process is similar no matter the entity type being sampled.
+
+To perturb PUMA, **50% of the time,** we replace the PUMA with a sample from
+an unconditional PUMA distribution; in other words, we use the PUMA value of
+a second independently sampled entity.
+If we are sampling from a subset of the ACS PUMS, for example recent immigrants
+consistent with a non-reference-person move, **80% of the perturbations**
```

Comment:
should this 80% be chosen uniformly at random?

also --- will the PUMA data that we sample from be the same data that we used for population initialization? are there any other sampling details that are important? do you expect all PUMAs to be represented in that data, for example? Or do you expect us to have to do any filtering to remove PUMAs that aren't residential? (im not sure if such a thing exists)

Reply:
Yes, uniform probability, and yes, sampling from the same data as initialization. I expect all PUMAs to be represented in that data.

There may (in real life) be PUMAs that skew heavily toward or away from GQ -- we'll capture this a bit through this "subset" concept that I clearly need to explain better, but otherwise we will not worry about it.